src/test/java/org/tugraz/sysds/test/functions/builtin/BuiltinScaleTest.java

```diff
@@ -0,0 +1,149 @@
+package org.tugraz.sysds.test.integration.functions.builtin;
+
+import java.util.HashMap;
+
+import org.junit.Test;
+import org.tugraz.sysds.common.Types.ExecMode;
+import org.tugraz.sysds.lops.LopProperties.ExecType;
+import org.tugraz.sysds.runtime.matrix.data.MatrixValue.CellIndex;
+import org.tugraz.sysds.test.AutomatedTestBase;
+import org.tugraz.sysds.test.TestConfiguration;
+import org.tugraz.sysds.test.TestUtils;
+
+public class BuiltinScaleTest extends AutomatedTestBase 
+{
+	private final static String TEST_NAME = "Scale";
+	private final static String TEST_DIR = "functions/builtin/";
+	private static final String TEST_CLASS_DIR = TEST_DIR + BuiltinScaleTest.class.getSimpleName() + "/";
```

Comment:
It doesn't really matter, but if would be nice to consistently use `private static final` instead of the different permutations of qualifiers.